test/fixtures/profiles/regressions/5131_input_equals/controls/input_equals.rb

```diff
@@ -0,0 +1,7 @@
+title 'Regressions for 5131'
```

Comment:
It's only a regression if it previously worked and we broke it. Did this case previously work OK in the past?

Reply:
When I said `Present from 4.18 onwards.` I meant it's known to be a problem that far back. I didn't mean that it worked previous to that version, sorry.

Reply:
Good point! I've reverted and tried to put this fixture in a more aptly named location. If the pipeline passes I'll give a rebase squash so we're clean again 👍

Reply:
Thanks. It would be fantastic to have a `regressions` directory for cases that are regressions. Sorry if my issue description was misleading